### PR TITLE
Ensure preview annotations are resolved relative to the cwd

### PR DESCRIPTION
**Pull request by tmeasday**
Issue: (when sandbox running in v6 mode, although this was a more general issue):

Pasted Graphic

## What I did

Make entry point filenames not have `/` in them.

## How to test

Run v6 sandbox. Note it won't run (2nd PR coming) but it should start.

### CODE REVIEW

File: code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts [hunk 1]
```diff
@@ -125,21 +125,24 @@ export default async (
       path.join(__dirname, '..', '..', 'templates', 'virtualModuleEntry.template.js')
     );
 
-    previewAnnotations.forEach((previewAnnotationFilename: any) => {
+    previewAnnotations.forEach((previewAnnotationFilename: string) => {
```
Comment (tmeasday) [inline code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts:128]:
```suggestion
    previewAnnotations.forEach((previewAnnotationFilename: string | undefined) => {
```

File: code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts [hunk 2]
```diff
@@ -125,21 +125,24 @@ export default async (
       path.join(__dirname, '..', '..', 'templates', 'virtualModuleEntry.template.js')
     );
 
-    previewAnnotations.forEach((previewAnnotationFilename: any) => {
+    previewAnnotations.forEach((previewAnnotationFilename: string | undefined) => {
+      if (!previewAnnotationFilename) return;
```
Comment (shilman) [inline code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts:129]: When does this exit early?

**Review by shilman — APPROVED** [review]
Can we get some kind of test for this?

File: code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts (hunk 2 above)
Comment (tmeasday) [reply code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts:129]: No idea. Previously it was just being cast to `any` and then used as a string

**Comment by tmeasday** [comment]
I don't think we have any meaningful tests for generated webpack configs, outside of running the Storybook. So the test would be getting a v6 store sandbox.

**Pull request merged**